confluence-syntax-xhtml/src/main/java/org/xwiki/contrib/confluence/parser/xhtml/internal/wikimodel/MacroParameterTagHandler.java

```diff
@@ -41,9 +41,18 @@
  */
 public class MacroParameterTagHandler extends AbstractMacroParameterTagHandler implements ConfluenceTagHandler
 {
+    protected static String IN_CONFLUENCE_PARAMETER = "in-confluence-parameter";
+    @Override
+    protected void begin(TagContext context)
+    {
+        super.begin(context);
+        context.getTagStack().pushStackParameter(IN_CONFLUENCE_PARAMETER, true);
```

Comment:
I'm wondering if it would not make more sense in AbstractMacroParameterTagHandler#begin/end.

Reply:
I don't know TaskStatusTagHandler and TaskId but what is sure is that it make sense for something called Abstract**MacroParameter**TagHandler.